Tools/electrostatic_sphere_plot.py

```diff
@@ -0,0 +1,149 @@
+import yt
```

Comment:
@ax3l @dbizzozero 
This file was useful in order to make some plots of the sphere ; however, it is not run in the automated tests - therefore I would suggest to remove it from this PR. 
(In general, we do *not* include plotting scripts in the examples, because plotting instructions are usually available in the WarpX documentation)